src/api/books/services/getBooks.ts

```diff
@@ -0,0 +1,11 @@
+import { DocumentType } from "@typegoose/typegoose";
+import { BookModel } from "../../../models";
+import Book from "../models/Book";
+
+export default async function getBooks(): Promise<DocumentType<Book>[]> {
+  try {
+    return await BookModel.find({});
```

Comment:
Guessing this will call the method that the search endpoint calls?

Reply:
No so this is just going to return all books. Not ideal long term but will work for demo home page